app/javascript/src/controller_services.js

```diff
@@ -889,9 +890,7 @@ function calculateAndCreatePageFlowConnectorPath(points, config) {
       }
     }
     else {
-      if(up) {
-        new ConnectorPath.ForwardDownBackwardUpPath(points, config);
-      }
```

Comment:
The `if(up)` here only drew the line if the 'to' FlowItem was in a row above the 'from' FlowItem. (see line 866)

This is not what we wanted. 

Because we always go `down` in order to go `back` we always need to go `up` regardless of what row the destination item is on.